index.html

```diff
@@ -2470,15 +2466,30 @@ <h2>Production and Consumption</h2>
 
       <ol>
         <li>
-A <a>conforming consumer</a> MUST add all entries that do not have explicit
-processing rules for the <a>representation</a> being consumed to the <a
-href="#data-model">data model</a> using only the <a>representation</a>'s data
-type processing rules.
+A <a>conforming consumer</a> MUST take a <a>representation</a> and
+MIME type <a data-cite="INFRA#string">string</a> as input into
+the <a>consumption</a> process. A <a>conforming consumer</a> MAY accept
+additional options as input into the <a>consumption</a> process.
         </li>
         <li>
 A <a>conforming consumer</a> MUST determine the <a>representation</a> of a
-<a>DID document</a> using the associated MIME type as described in <a
-href="#did-resolution-metadata"></a>.
+<a>DID document</a> using the MIME type input <a
+data-cite="INFRA#string">string</a>.
+        </li>
+        <li>
+A <a>conforming consumer</a> MUST detect any representation-specific
+entry across all known <a>representations</a> and place the entry into a
+representation-specific entries <a data-cite="INFRA#maps">map</a> which is
```

Comment:
I had played around with alternative language here that effectively said: "any representation specific entries that could appear in the representation" -- which basically boils down to just `@context`... `%YAML` and `xmlns` are highly unlikely to appear in JSON and CBOR.